docs/installation/installing-with-npm.md

```diff
@@ -106,45 +103,124 @@ $govuk-global-styles: true;
 @import "govuk-frontend/frontend/all";
 ```
 
-### Importing JavaScript
+## Using JavaScript
 
 Some of the JavaScript included in GOV.UK Frontend improves the usability and
-accessibility of the components. You should make sure that you are importing and
-initialising Javascript in your application to ensure that all users can use it successfully.
+accessibility of the components.
 
 For example, the JavaScript will:
 
 - allow links styled as buttons to be triggered with the space bar when focused,
   which matches the behaviour of native buttons and the way the button is
   described when using assistive technologies.
-- enhance the Details component to help users of assistive technologies
+- enhance the details component to help users of assistive technologies
   understand whether it is expanded or collapsed, and to make the component
   behave correctly for users of Internet Explorer 8.
 
-You can include and initialise the Javascript for all components by adding an import statement to your application's main JavaScript file:
+You should [include](#option-1-include-javascript) or [import](#option-2-import-javascript) GOV.UK Frontend JavaScript, and then initialise the script in your application to ensure that all users can use it successfully. Note that GOV.UK Frontend does not initialise any scripts by default; all scripts **must** be initialised in order for them to work.
+
+### Option 1: Include JavaScript
+
+Include the `node_modules/govuk-frontend/all.js` script on your page. You might wish to copy the file into your project or reference it from `node_modules`.
+
+See below for an example:
+
+```html
+<script src="govuk-frontend/all.js"></script>
+```
+
+#### Initialise individual included components
+
+Once you have included the script, you can initialise individual components using the `data-module` attribute set on the component. If you only have one instance of a single component per page, you can follow the below example to initialise it:
 
 ```js
-import All from 'govuk-frontend/all'
+var Radios = window.GOVUKFrontend.Radios
+var $radio = document.querySelector('[data-module="radios"]')
+if ($radio) {
+  new Radios($radio).init()
+}
 ```
 
-Alternatively, you can import and initialise individual components, such as the
-Button component:
+If you have more than one instance of a component per page, you will need to initialise each one separately.
+
+The button component is initialised differently as the script polyfills globally for all anchor tags with `role="button"`; for this reason, there is no need to initialise using the `data-module` attribute. You can initialise the button component with the below:
 
 ```js
-import Button from 'govuk-frontend/components/button/button'
+var Button = window.GOVUKFrontend.Button
+new Button(document).init()
+```
 
-new Button().init()
+#### Initialise all included components
```

Comment:
I think this would make sense to be above the other, since I imagine most people will use this by default.

Reply:
We could consider merging this example with the place we tell people to include a script tag.